client/src/containers/App.tsx

```diff
@@ -264,7 +264,17 @@ const Navbar = () => {
 
   const userContext = useContext(UserContext);
 
-  return (
+  const standalonePages = [
+    "forgot-password",
+    "login",
+    "verify-email",
+    "forgot-password",
+    "reset-password",
+    "unsubscribe",
+  ];
+  const hideNavbar = standalonePages.some((v) => pathname.includes(`account/${v}`));
+
+  return hideNavbar ? null : (
```

Comment:
hmm i hear ya. i can't think of a better way to do this logically but there is a way to avoid returning a null, like so. tried it locally and it works!

```
return (
  <>
    {!hideNavbar && (insert JSX element here)}
  </>
);
```